### PR TITLE
Enable signing of artifacts larger than 1MiB

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None

Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

File: awriter/signer.go [hunk 1]
```diff
@@ -61,18 +60,9 @@ func SignExisting(src io.Reader, dst io.Writer, key []byte, overwrite bool) erro
...
 
-		read, err := rTar.Read(buf)
-		if err != nil && err != io.EOF {
-			return errors.Wrap(err, "Could not read tar file entry")
-		} else if read == 0 {
-			continue
-		}
-
-		written, err := wTar.Write(buf[:read])
+		_, err = io.Copy(wTar, rTar)
```
Comment (oleorhagen) [inline awriter/signer.go]: Better to use the io.Copy functionality than basically juggle readers and writers ourself imo.

Comment (kacf) [reply awriter/signer.go]: :+1:

**Comment by oleorhagen** [comment]
@kacf, It fails the integration tests with the issues we've been seeing lately. Also, the codecov patch is a little weird. 
Is it good to go you reckon?

**Comment by kacf** [comment]
Yes, nothing here I haven't already seen elsewhere. Good to go as far as I'm concerned!

**Pull request merged**